explore-analyze/transforms/transform-limitations.md

```diff
@@ -123,6 +123,55 @@ If your data uses the [date nanosecond data type](elasticsearch://reference/elas
 
 If you use ILM to have time-based indices, consider using the [Date index name](elasticsearch://reference/enrich-processor/date-index-name-processor.md) instead. The processor works without duplicated documents if your {{transform}} contains a `group_by` based on `date_histogram`.
 
+### SLO transforms at scale may misreport state and increase PIT activity
+
+When running a large number of SLO {{transforms}}, two types of limitations can appear:
+
+- {{transforms-cap}} may [misreport](#transforms-inaccurate-errors) their state and suggest deletion even though they are still valid
+- Point-in-time (PIT) requests may [accumulate](#transforms-pit-storms), creating extra activity on the cluster and slowing down operations
+
+#### {{transforms-cap}} can return inaccurate errors that suggest deletion [transforms-inaccurate-errors]
+
+The {{transforms-cap}} API and the {{transforms-cap}} page in {{kib}} (**Stack Management** > **{{transforms-cap}})** may display misleading error messages for {{transforms}} created by service level objectives (SLOs).
+
+The message typically reads:
+
+```console
+Found task for transform [...], but no configuration for it. To delete this transform use DELETE with force=true.
+```
+Although the error suggests that the {{transform}} is broken and should be deleted, this is usually not the case. The error is caused by a bug in how the {{transforms}} are queried and presented by the API and UI. Deleting the {{transform}} can unnecessarily break related SLOs.
```

Comment:
```suggestion
Although the error suggests that the {{transform}} is broken and should be deleted, this is usually not the case. The error occurs because of a limitation in how the {{transforms}} are queried and presented by the API and UI. Deleting the {{transform}} can break related SLOs.
```